src/irr/asset/CImageLoaderJPG.cpp

```diff
@@ -267,8 +268,12 @@ asset::IAsset* CImageLoaderJPG::loadAsset(io::IReadFile* _file, const asset::IAs
 //				data[i+1] = output[j+1];
 //				data[i+2] = output[j+0];
 				data[i+0] = (char)(output[j+2]*(output[j+3]/255.f));
-				data[i+1] = (char)(output[j+1]*(output[j+3]/255.f));
+				os::Printer::log(std::to_wstring(data[i + 0]), ELL_DEBUG);
+				data[i + 1] = (char)(output[j + 1] * (output[j + 3] / 255.f)); const char d = data[i + 1];
+				os::Printer::log(std::to_wstring(data[i + 1]), ELL_DEBUG);
```

Comment:
A. Don't log the pixels!
B. Now that sRGB has been set as format, remove this squaring conversion!